src/pages/rest/b2b/negotiable-quote.md

```diff
@@ -19,6 +19,13 @@ The quoting process itself can be a continuous process, with a number of repeati
 -  The buyer reviews the seller's counteroffer
 -  Upon agreement, the buyer begins the checkout process and the system converts the negotiable quote into an order
 
+A negotiable quote can also be initiated by a seller. In this case, the negotiation lifecycle includes the following stages:
+
+- The seller creates and submits a negotiable quote
+- The buyer reviews and modifies the quote
```

Comment:
```suggestion
- The buyer reviews and modifies the quote or begins the checkout process
```

Reply:
Thats description of negotiation cycles. Proposed addition covered in "Upon agreement, the buyer begins the checkout process and the system converts the negotiable quote into an order"